### PR TITLE
Implement create world

### PR DESCRIPTION
**Pull request by PeachesMLG**
# Description
State the changes of the pull request here.

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Version number updated in `settings.gradle`.
- [ ] Unit tests added.
- [ ] Code follows existing code format.

# Info on creating a pull request
Before the pull request can be accepted, the version number stored in settings.gradle should be updated.
MockBukkit uses semantic-versioning (https://semver.org/)

 - If the PR fixes a bug in MockBukkit, update the patch version. (if the current version is `0.4.1`, the new version should be `0.4.2`)
 - If the PR adds a new feature to MockBukkit, update minor version. (if the current version is `0.4.1`, the new version should be `0.5.0`)
 - Make sure that unit tests are added which test the relevant changes.
 - Make sure that the changes follow the existing code format.

Note that a PR that fixes an `UnimplementedOperationException` should be considered a new version and not a bugfix.

The version number can be found in `settings.gradle`.

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/ServerMock.java [hunk 1]
```diff
@@ -994,8 +994,7 @@ public ServicesManagerMock getServicesManager()
 	@Override
 	public World createWorld(WorldCreator creator)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return addSimpleWorld(creator.name());
```
Comment (TheBusyBiscuit) [inline src/main/java/be/seeseemelk/mockbukkit/ServerMock.java]: The point of a `WorldCreator` is to create a World that fits certain requirements.
These requirements include:
* Name
* Seed
* Environment
* ChunkGenerator
* BiomeProvider
* WorldType
* GenerateStructures (boolean)
* GeneratorSettings
* HardcoreMode (boolean)

Your implementation only fulfills one of these requirements, if I want to create a Nether world and I get an overworld, clearly something's wrong.

If you are interested in a proper implementation of this, I suggest you to take a test-driven approach in this case.
Write a Test class for the `WorldCreator` and write test cases for each of these requirements.
Set one of those values, create a world and check if the requirement was fulfilled.
Then implement accordingly.

Of course we don't need actually need chunk generation or biomes and stuff here.
But environment, seed, worldtype and such should be the minimum here.
The rest can always be implemented later (if anyone cares for those features)

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/be/seeseemelk/mockbukkit/ServerMock.java (hunk 1 above)
Comment (PeachesMLG) [reply src/main/java/be/seeseemelk/mockbukkit/ServerMock.java]: I can add mocked Environment, but both getSeed and getWorldType return unimplemented exceptions, so does anyone even care about that?

**Comment by PeachesMLG** [comment]
uh... Think I had some issues with versions or something, seems some of the code rolled back, ima make another pr

File: src/main/java/be/seeseemelk/mockbukkit/ServerMock.java (hunk 1 above)
Comment (TheBusyBiscuit) [reply src/main/java/be/seeseemelk/mockbukkit/ServerMock.java]: In terms of simple get operations, yes. Those should be covered.

**Pull request closed without merging**